config.js

```diff
@@ -45,7 +45,8 @@ const ConfigOptions = () => {
             },
             jsDir: 'js',
             lintPaths: [''],
-            usePackageVersion: false
+            usePackageVersion: false,
+            stripDebug: true
```

Comment:
Can you add the relevant unit tests to `config.test.js` please?

Reply:
Done in latest commit 👍